### PR TITLE
Add `(s)` to the apply command

### PR DESCRIPTION
**Pull request by ThomasLaPiana**
Closes n/a

### Code Changes

* [x] Update the language in `apply.py` so that it is plural/singular agnostic

### Steps to Confirm

* [x] run apply commands locally to confirm this works

### Pre-Merge Checklist

* [ ] All CI Pipelines Succeeded
* [ ] Documentation Updated
* [ ] Issue Requirements are Met
* [ ] Relevant Follow-Up Issues Created

### Description Of Changes

_Write some things here about the changes and any potential caveats_

### CODE REVIEW

File: fidesctl/src/fidesctl/core/apply.py [hunk 1]
```diff
@@ -64,7 +64,7 @@ def echo_results(action: str, resource_type: str, resource_count: int) -> None:
     """
     Echo out the results of the apply.
     """
-    echo_green(f"{action.upper()} {resource_count} {resource_type} resources.")
+    echo_green(f"{action.upper()} {resource_count} {resource_type} resource(s).")
```
Comment (PSalant726) [inline fidesctl/src/fidesctl/core/apply.py:67]: I forgot we have this helper here, so I think it would be pretty easy to dynamically pluralize the text:
```suggestion
	pluralize = "s" if resource_count != 1 else ""
    echo_green(f"{action.upper()} {resource_count} {resource_type} resource{pluralize}.")
```

File: fidesctl/setup.py [hunk 2]
```diff
@@ -20,15 +20,15 @@
 redshift_connector = "sqlalchemy-redshift==0.8.8"
 fastapi = "fastapi==0.68"
 uvicorn = "uvicorn==0.15"
-aws_connector = "boto3==1.20.54"
+aws_connector = "boto3==1.21.33"
```
Comment (PSalant726) [inline fidesctl/setup.py:23]: IMO the changes in this file really belong in their own PR. Bumping a dependency version should be documented/justified.

**Review by PSalant726 — APPROVED** [review]
Thanks for this, it really soothes my OCD 😅 

Nothing blocking, just some small suggestions.

File: fidesctl/src/fidesctl/core/apply.py (hunk 1 above)
Comment (ThomasLaPiana) [reply fidesctl/src/fidesctl/core/apply.py:67]: This just feels like too much....`(s)` is a well understood shorthand for this exact situation
